packages/hw-app-ada/src/Ada.js

```diff
@@ -79,15 +100,25 @@ export default class Ada {
     await this.transport.send(CLA, INS_RUN_TESTS, 0x00, 0x00);
   }
 
-  async attestUTxO(
+  /**
+   * Attests the utxo.
+   *
+   * @returns {Promise<{txHash:string, outputNumber:number, amount:number, hmac:string}>}.
+   */
+  async attestUtxo(
     txHex: string,
     outputIndex: number
-  ): Promise<{ amount: number }> {
+  ): Promise<{
+    txHash: string,
+    outputNumber: number,
+    amount: number,
+    hmac: string
+  }> {
     const P1_INIT = 0x01;
     const P1_CONTINUE = 0x02;
     const P2_UNUSED = 0x00;
 
-    const CHUNK_SIZE = 255;
+    const CHUNK_SIZE = 100;
```

Comment:
Why are we changing this constant?

Reply:
It stopped working at the time you fiddled with stream size. So I lowered it and it works again.

Reply:
If necessary, I might try to find higher working value.

Reply:
Yep, it works. Sorry, must have been that I tried it with stream size 100 on Ledger and than associated it with new change (not with my change).